orchestrator/api/api_v1/endpoints/settings.py

```diff
@@ -54,6 +54,12 @@ def get_cache_names() -> dict[str, str]:
     return CACHE_FLUSH_OPTIONS
 
 
+@router.post("/search-index/reset")
+async def reset_search_index() -> None:
+    settings.reset_search_index()
+    return
```

Comment:
this should also work
```suggestion
   return settings.reset_search_index()
```